msbuild/Xamarin.Mac.Tasks/Xamarin.Mac.ObjCBinding.CSharp.targets

```diff
@@ -67,15 +67,15 @@ Copyright (C) 2014 Xamarin Inc. All rights reserved.
 			AllowUnsafeBlocks="$(AllowUnsafeBlocks)"
 			ApiDefinitions="@(ObjcBindingApiDefinition)"
 			BaseLibDll="$(BaseLibDllPath)"
-			CompilerPath="$(CscPath)"
+			CompilerPath="/Library/Frameworks/Mono.framework/Commands/csc"
 			CoreSources="@(ObjcBindingCoreSource)"
 			DefineConstants="$(DefineConstants)"
 			EmitDebugInformation="$(BTouchEmitDebugInformation)"
 			ExtraArgs="$(BTouchExtraArgs)"
 			GeneratedSourcesDir="$(GeneratedSourcesDir)"
 			GeneratedSourcesFileList="$(_GeneratedSourcesFileList)"
 			Namespace="$(Namespace)"
-			NoStdLib="$(NoStdLib)"
+			NoStdLib="True"
```

Comment:
that one make sense (to me) since it's something we *want* (and need) to enforce, i.e. we are the one to override anything from above